src/Tizen.Multimedia.MediaPlayer/Player/Player.Events.cs

```diff
@@ -74,6 +74,13 @@ public partial class Player
         public event EventHandler<BufferingProgressChangedEventArgs> BufferingProgressChanged;
         private NativePlayer.BufferingProgressCallback _bufferingProgressCallback;
 
+        /// <summary>
+        /// Occurs when the player is prepared.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<EventArgs> Prepared;
```

Comment:
Did you intend to just notify `preparing is done` to user?
If so, I think it doesn't needed. 
The usage of this method is the same with existing `PrepareAsync()` method.
I think the only difference is that `Cancel` is available or not.

Reply:
I also agree to hsgwon.

Reply:
ok i'll delete it.